wordpress-mobile/WordPress-iOS
# Blaze Manage Campaigns: Add Blaze campaigns dashboard card

Related Issue: #20745

This PR adds the views, but none of the backend logic (is TBD) or user interactions yet.

To test:

1. In DashboardCard.swift replace `DashboardBlazeCardCell.self` with `DashboardBlazeCampaignCardCell.self`
2. Open dashboard on any site that supports Blaze and verify that the recent (mocked) campaign is displayed, along with its stats, and the button to show the remaining campaigns

<img width="302" alt="Screenshot 2023-06-07 at 5 04 12 PM" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/baf6da8b-b231-4af8-84e2-e4dcdb41be04"> <img width="302" alt="Screenshot 2023-06-07 at 5 06 49 PM" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/51c62e91-f923-4977-aafe-ac29d42e2497">

## Regression Notes
1. Potential unintended areas of impact: n/a (the card isn't fully integrated into the app yet)
4. What I did to test those areas of impact (or what existing automated tests I relied on): test manually
5. What automated tests I added (or what prevented me from doing so): ViewModel unit tests

PR submission checklist:
